src/edu/washington/escience/myria/parallel/Worker.java

```diff
@@ -159,13 +158,20 @@ public void run() {
                   abruptShutdown = false;
                   break;
                 case ADD_WORKER:
+                  int workerId = cm.getWorkerId();
                   if (LOGGER.isInfoEnabled()) {
                     LOGGER.info("received ADD_WORKER " + workerId);
                   }
                   connectionPool.putRemote(
                       workerId, SocketInfo.fromProtobuf(cm.getRemoteAddress()));
                   sendMessageToMaster(IPCUtils.addWorkerAckTM(workerId));
                   break;
+                case SYSTEM_GC:
+                  if (LOGGER.isInfoEnabled()) {
+                    LOGGER.info("Trigger System GC");
```

Comment:
Given that we're not computing anything here, not sure why conditional is necessary...